spec/helpers/email_confirmation_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the EmailConfirmationHelper. For example:
+#
+# describe EmailConfirmationHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe EmailConfirmationHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
It's already a time to have specs for your code.

Reply:
What do you mean?

Reply:
Tests for: Models and Service objects